packages/starlight/schema.ts

```diff
@@ -142,6 +142,12 @@ export function docsSchema() {
 					 * Defaults to the page `title` if not set.
 					 */
 					label: z.string().optional(),
+
+					/**
+					 * Hides the page from any navigation that is autogenerated
+					 * Ignored if explicitly referenced in the sidebar config
```

Comment:
I think we’re OK to leave out the second line here as we’re already saying when this applies in the first.

```suggestion
					 * Prevents this page from being included in autogenerated sidebar groups.
```